classes/class-wc-connect-help-view.php

```diff
@@ -73,26 +73,8 @@ protected function get_health_items() {
 			// Only one of the following should present
 			// Check that Jetpack is active
 			// Check that Jetpack is connected
-			include_once ABSPATH . 'wp-admin/includes/plugin.php'; // required for is_plugin_active
 			$is_connected = WC_Connect_Jetpack::is_active() || WC_Connect_Jetpack::is_development_mode();
-			if ( ! is_plugin_active( 'jetpack/jetpack.php' ) ) {
```

Comment:
No need for all this branching logic anymore - checking whether jetpack is installed or not won't matter, we are now using the composer packages directly.